trlx/utils/loading.py

```diff
@@ -14,6 +9,11 @@
 from trlx.pipeline import _DATAPIPELINE
 from trlx.pipeline.offline_pipeline import PromptPipeline
 
+# Register load models via module import
```

Comment:
Somtimes isort fucks up files like this. Maybe loading should have isort skip?

Reply:
I actually think it's doing the right thing here since, previously these were `.models` imports and never were re-sorted as `.trainers`. But I can skip this file if you think it's best. Let me know!